### PR TITLE
feat: `releaseNameTemplate` and `releaseBodyTemplate` options

### PR DESCRIPTION
**Pull request by halkeye**
I want to be able to be able to add some installation instructions and other preambles to each release. This seemed like the simplest way to do that, as well as handle #498

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,6 +93,8 @@ When using the _GITHUB_TOKEN_, the **minimum required permissions** are:
...
 | `draftRelease`        | A boolean indicating if a GitHub Draft Release should be created instead of publishing an actual GitHub Release.                                                                                       | `false`                                                                                                                                              |
+| `releaseName`         | A [Lodash template](https://lodash.com/docs#template) to customize the github release's name                                                                                                           | `<%= nextverison.name %>`                                                                                                                            |
+| `releaseBody`         | A [Lodash template](https://lodash.com/docs#template) to customize the github release's body                                                                                                           | `<%= nextverison.notes %>`                                                                                                                           |
```
Comment (gr2m) [inline README.md:97]: should we rename the arguments to `releaseNameTemplate`/`releaseBodyTemplate` for clarity?

**Review by gr2m — APPROVED** [review]
great pull request, thank you! I'm okay to merge as is but have a suggestion

File: README.md (hunk 1 above)
Comment (halkeye) [reply README.md:97]: Good call, updated

**Review by gr2m — APPROVED** [review]
Thank you!

**Comment by halkeye** [comment]
I stupidly clicked the rebase with master, so CI will need to be re-approved sorry

**Review by gr2m — APPROVED** [review]

**Pull request merged**